kolibri/plugins/device_management/assets/src/views/ManageContentPage/SelectNetworkAddressModal/SelectAddressForm.vue

```diff
@@ -0,0 +1,224 @@
+    >
+      {{ uiAlertProps.text }}
+      <KButton
+        v-if="requestsFailed"
+        appearance="basic-link"
+        :text="$tr('refreshAddressesButtonLabel')"
+        @click="refreshAddressList"
+      />
+    </UiAlert>
+
+    <template v-for="(a, idx) in addresses">
+      <div :key="`div-${idx}`">
+        <KRadioButton
+          class="radio-button"
+          v-model="selectedAddressId"
+          :key="idx"
+          :value="a.id"
+          :label="a.device_name"
+          :description="a.base_url"
+          :disabled="!a.available || !a.hasContent"
+        />
+        <KButton
+          @click="removeAddress(a.id)"
+          :key="`forget-${idx}`"
+          :text="$tr('forgetAddressButtonLabel')"
+          appearance="basic-link"
+        />
+      </div>
+    </template>
+  </KModal>
+
+</template>
+
+
+<script>
+
+  import { mapGetters, mapState } from 'vuex';
+  import find from 'lodash/find';
+  import KButton from 'kolibri.coreVue.components.KButton';
+  import KModal from 'kolibri.coreVue.components.KModal';
+  import KRadioButton from 'kolibri.coreVue.components.KRadioButton';
+  import UiAlert from 'keen-ui/src/UiAlert';
+  import { deleteAddress, fetchAddresses } from './api';
+
+  const Stages = {
+    FETCHING_ADDRESSES: 'FETCHING_ADDRESSES',
+    FETCHING_SUCCESSFUL: 'FETCHING_SUCCESSFUL',
+    FETCHING_FAILED: 'FETCHING_FAILED',
+    DELETING_ADDRESS: 'DELETING_ADDRESS',
+    DELETING_SUCCESSFUL: 'DELETING_SUCCESSFUL',
+    DELETING_FAILED: 'DELETING_FAILED',
+  };
+
+  export default {
+    name: 'SelectAddressForm',
+    components: {
+      KButton,
+      KModal,
+      KRadioButton,
+      UiAlert,
+    },
+    props: {},
+    data() {
+      return {
+        addresses: [],
+        selectedAddressId: '',
+        showUiAlerts: false,
+        stage: '',
+        Stages,
+      };
+    },
+    computed: {
+      ...mapGetters('manageContent/wizard', ['isImportingMore']),
+      ...mapState('manageContent/wizard', ['transferredChannel']),
+      submitDisabled() {
+        return (
+          this.selectedAddressId === '' ||
+          this.stage === Stages.FETCHING_ADDRESSES ||
+          this.stage === Stages.DELETING_ADDRESS
+        );
+      },
+      newAddressButtonDisabled() {
+        return this.stage === Stages.FETCHING_ADDRESSES;
+      },
+      requestsSucessful() {
+        return (
+          this.stage === Stages.FETCHING_SUCCESSFUL || this.stage === Stages.DELETING_SUCCESSFUL
+        );
+      },
+      requestsFailed() {
+        return this.stage === Stages.FETCHING_FAILED || this.stage === Stages.DELETING_FAILED;
+      },
+      uiAlertProps() {
+        if (this.stage === Stages.FETCHING_ADDRESSES) {
+          return {
+            text: this.$tr('fetchingAddressesText'),
+            type: 'info',
+          };
+        }
+        if (this.requestsSucessful && this.addresses.length === 0) {
+          return {
+            text: this.$tr('noAddressText'),
```

Comment:
I don't think we should use a ui-alert for this empty state.

Reply:
@khangmach @jtamiace

Reply:
I'm just copying the UI-alert found when no export drives found/no import locations found. Open to changes though

Reply:
Yeah, it is analogous to an empty drive list in the 'Select a drive' Modal

Reply:
:arthur-fist: